3rdparty/libunwind/CMakeLists.txt

```diff
@@ -45,7 +45,6 @@ add_library(libunwind OBJECT
   libunwind/src/dwarf/Lfind_unwind_table.c
   libunwind/src/dwarf/Lparser.c
   libunwind/src/dwarf/Lpe.c
-  libunwind/src/dwarf/Lstep.c
```

Comment:
@HernanGatta Do the OE unwind tests pass?
Since unwinding has to stop at the enclave boundary for SGX, OE has this implementation:
https://github.com/microsoft/openenclave/blob/master/3rdparty/libunwind/Gstep.c